src/libs/rpc.ts

```diff
@@ -0,0 +1,9 @@
+export default function getRpcUrl(networkId: number): string | undefined {
+  switch (networkId) {
+    case 1:
+      return process.env.MAINNET_RPC;
```

Comment:
We should have a `.env.example` with all these variables!

Reply:
Fixed in 4e7b000ac48a48779622f01b7a40c52c35a50f45